packages/websites-server/scripts/clear-cloudfront-cache.js

```diff
@@ -0,0 +1,49 @@
+const {
+  CloudFrontClient,
+  CreateInvalidationCommand
+} = require('@aws-sdk/client-cloudfront')
+
+module.exports = async function () {
+  const {
+    CLOUDFRONT_DISTRIBUTION_ID,
+    CONTEXT,
+    DVCORG_AWS_ACCESS_KEY_ID,
+    DVCORG_AWS_SECRET_ACCESS_KEY
+  } = process.env
+  if (CONTEXT !== 'production') {
+    console.log(
+      'Skipping CloudFront cache purge because CONTEXT is not set to production.'
+    )
+    return
+  }
+
+  if (!CLOUDFRONT_DISTRIBUTION_ID) {
+    console.error(
+      'Skipping CloudFront cache purge because CLOUDFRONT_DISTRIBUTION_ID is not set.'
+    )
+    return
+  }
+
+  const client = new CloudFrontClient({
+    region: 'us-east-1',
+    credentials: {
+      accessKeyId: DVCORG_AWS_ACCESS_KEY_ID,
+      secretAccessKey: DVCORG_AWS_SECRET_ACCESS_KEY
```

Comment:
I will remove this prefix once we rename `dvc-public`.

Reply:
https://github.com/iterative/dvc.org/blob/7222ff3237fe9fd41c8dd578735b36927391d6fa/packages/websites-server/scripts/s3-utils.js#L13

Why introduce separate variable when there is already one ? @skshetry